svgdigitizer/__main__.py

```diff
@@ -21,7 +21,7 @@
 # ********************************************************************
 import click
 
-help_sampling = 'sampling interval (in mV)'
+help_sampling = 'sampling interval on the x-axis'
```

Comment:
The unit is now in the unit that is specified in the plot. So if the axis is in mV nothing changes.